boards/opentitan/src/tests/hmac.rs

```diff
@@ -111,20 +121,22 @@ fn hmac_check_verify() {
     let hmac = &perf.hmac;
 
     let callback = unsafe { static_init_test_cb() };
-    let buf = LeasableBuffer::new(callback.input_buffer.take().unwrap());
+
+    let _buf = LeasableMutableBuffer::new(callback.input_buffer.take().unwrap());
```

Comment:
```suggestion
    callback.input_buffer.take().unwrap();
```
it seems like the construction of the leasable buffer is unneeded?